src/Test.elm

```diff
@@ -0,0 +1,303 @@
+module Test exposing (Test, FuzzOptions, describe, test, filter, concat, fuzz, fuzz2, fuzz3, fuzz4, fuzz5, fuzzWith)
+
+{-| Writing tests.
+
+@docs Test, test
+
+## Organizing Tests
+
+@docs describe, concat, filter
+
+## Fuzz Testing
+
+@docs fuzz, fuzz2, fuzz3, fuzz4, fuzz5, fuzzWith, FuzzOptions
+-}
+
+import Random
+import Test.Internal as Internal
+import Expect exposing (Expectation)
+import Random exposing (Generator)
+import Fuzz exposing (Fuzzer)
+
+
+{-| A test which has yet to be evaluated. When evaluated, it produces one
+or more [`Expectation`](../Expect#Expectation)s.
+
+See [`test`](#test) and [`fuzz`](#fuzz) for some ways to create a `Test`.
+-}
+type alias Test =
+    Internal.Test
+
+
+{-| Run the given tests in order.
+
+    concat [ testDecoder, testSorting ]
+-}
+concat : List Test -> Test
+concat =
+    Internal.Batch
+
+
+{-| Remove any test unless its description satisfies the given predicate
+function. Nested descriptions added with [`describe`](#describe) are not considered.
+-}
```

Comment:
Yeah I think an example is necessary...thanks for reminding me though - I'd forgotten about this TODO. 😄